modules/oe_theme_helper/src/Plugin/Block/PageHeaderBlock.php

```diff
@@ -99,4 +124,32 @@ public function setTitle($title): self {
     return $this;
   }
 
+  /**
+   * Returns a list of breadcrumb items ready to be rendered.
+   *
+   * @param string $title
+   *   The title of the current page.
+   *
+   * @return array
+   *   A list of breadcrumb items.
+   */
+  protected function getBreadcrumbSegments($title): array {
+    $breadcrumb = $this->breadcrumbBuilder->build($this->currentRouteMatch);
```

Comment:
The `\Drupal\Core\Breadcrumb\Breadcrumb` class has cacheability metadata. We need to bubble up that data. See `::toRenderable()` in said class.